ResearchKit/Consent/ORKConsentSceneViewController.m

```diff
@@ -76,12 +76,26 @@ - (void)setConsentSection:(ORKConsentSection *)consentSection {
     self.headerView.captionLabel.text = consentSection.title;
     
     self.imageView.image = consentSection.image;
-    self.headerView.instructionLabel.text = [consentSection summary];
+    self.headerView.instructionLabel.attributedText = [self getAttributedSummary:consentSection.summary];
     
     self.continueSkipContainer.continueEnabled = YES;
     [self.continueSkipContainer updateContinueAndSkipEnabled];
 }
 
+- (NSAttributedString *)getAttributedSummary:(NSString *)summary {
+    NSString *openFont = @"<font size='5' style='font-family:HelveticaNeue-Light'>";
+    NSString *closeFont = @"</font>";
+    NSString *customTitle = [NSString stringWithFormat:@"%@%@%@", openFont, summary, closeFont];
+    NSError *error = nil;
+    NSMutableAttributedString *attts = [[NSMutableAttributedString alloc] initWithData:[customTitle dataUsingEncoding:NSUnicodeStringEncoding] options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:nil error:&error];
+    NSRange range = NSMakeRange(0, attts.length);
+    NSMutableParagraphStyle *style =  [[NSMutableParagraphStyle alloc] init];
+    style.alignment = NSTextAlignmentCenter;
+    [attts addAttribute:NSParagraphStyleAttributeName value:style range:range];
+    
+    return attts;
+}
+
```

Comment:
Better :), event though if there's time I'd probably call the getAttributtedText withFontSize 5 from this instead of having the code duplicated for this one.

Reply:
Fixed :)